windows/deploy_vm/deploy_vm_from_iso.yml

```diff
@@ -77,19 +77,34 @@
   vars:
     vm_power_state_set: "powered-on"
 
-- name: "Send key strokes to VM console"
-  include_tasks: ../../common/vm_guest_send_key.yml
-  vars:
-    keys_send:
-      - DOWNARROW
-      - DOWNARROW
-      - ENTER
-      - ENTER
+- name: "VM firmware is EFI"
   when: firmware | lower == "efi"
+  block:
+    - name: "Pause 5 seconds when VM power on"
+      ansible.builtin.pause:
+        seconds: 5
+    - name: "Take screenshot after VM power on"
+      include_tasks: ../../common/vm_take_screenshot.yml
+      vars:
+        vm_screenshot_local_dir: "{{ current_test_log_folder }}"
+        vm_screenshot_local_name: "after_power_on.png"
```

Comment:
Please add `screenshot_` prefix so that it can be easily recognized in test case log folder.